app/forms/assets/AssetDescriptionFormProvider.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.assets
+
+import forms.Validation
+import forms.mappings.Mappings
+import javax.inject.Inject
+import play.api.data.Form
+
+class AssetDescriptionFormProvider @Inject() extends Mappings {
+  val maxLengthClassOfBeneficiary = 70
```

Comment:
This is incorrect. Schema says business asset description is 56. Easiest thing to do would be to remove this file and use the DescriptionFormProvider, where you can pass the message key prefix and maximum description length into it